Parser/pgen/keywordgen.py

```diff
@@ -0,0 +1,62 @@
+"""Generate Lib/keyword.py from the Grammar and Tokens files using pgen"""
+
+import argparse
+
+from .pgen import ParserGenerator
+
+TEMPLATE = '''\
+#! /usr/bin/env python3
+
+"""Keywords (from "Grammar/Grammar")
+
+This file is automatically generated; please don't muck it up!
+
+To update the symbols in this file, 'cd' to the top directory of
+the python source tree and run:
+
+    python -m Parser.pgen.keywordgen ./Grammar/Grammar \\
+                                     ./Grammar/Tokens \\
+                                     ./Lib/keyword.py
+
+Alternatively, you can run 'make regen-keyword'.
+"""
+
+__all__ = ["iskeyword", "kwlist"]
+
+kwlist = [
+{keywords}
+        ]
+
+iskeyword = frozenset(kwlist).__contains__
+'''
+
+EXTRA_KEYWORDS = ["async", "await"]
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Parser generator main program.")
+    parser.add_argument(
+        "grammar", type=str, help="The file with the grammar definition in EBNF format"
+    )
+    parser.add_argument(
+        "tokens", type=str, help="The file with the token definitions"
+    )
+    parser.add_argument(
+        "keyword_file",
+        type=argparse.FileType('w'),
+        help="The path to write the keyword definitions",
+    )
+    args = parser.parse_args()
+    p = ParserGenerator(args.grammar, args.tokens)
+    grammar = p.make_grammar()
+
+    with args.keyword_file as thefile:
+        all_keywords = sorted(list(grammar.keywords) + EXTRA_KEYWORDS)
+
+        keywords = "\n".join("        '{}',".format(keyword)
+                             for keyword in all_keywords)
```

Comment:
While we are modifying keyword.py... can I suggest a minor coding style change?

I propose to use ",\n    ".join("'{}'".format(keyword) for keyword in all_keywords)) to avoid the final ",".

And then modify the template:

```

kwlist = [
    {keywords}]
```

It should give indent values to 4 spaces like that:

```
keywords = [
    'a',
    'b',
    'c']
```

Reply:
Done in bffdb57 :D